### PR TITLE
Find ngos

### LINKED ISSUE #62: Resources Page
Create the resources page requirements 
* Connected to firestore
* Filter and search
* Redirecting the user to the resource that they choose
* connect it to google API

### PR DESCRIPTION
**Pull request by fadime-ozdemir**
Functionalities for the resources page
- search by keywords
- filter by options

### CODE REVIEW

File: src/components/AdvancedFilters/index.jsx [hunk 1]
```diff
@@ -7,14 +7,43 @@ import {
...
+            chosenOptions={chosenOptions}
+            chosenOptionsKey={option}
+          />
+        );
+      }
+      return null;
+    });
+  };
+  console.log("options  =>", filterOptions);
+  console.log("selectedOptions===>", chosenOptions);
```
Comment (khaldoon9kh) [inline src/components/AdvancedFilters/index.jsx]: remove console.log

File: src/components/FilterOptionsList/index.jsx [hunk 2]
```diff
@@ -3,14 +3,21 @@ import "./index.css";
...
-      {filter.filterParameters.map((option) => {
+      {filter.map((option) => {
         return (
           <div
             className="filterOption d-flex align-items-center p-1"
             key={option}
+            onClick={() =>
+              setChosenOptions((state) => {
+                console.log(state);
+                console.log(filterCategory);
```
Comment (khaldoon9kh) [inline src/components/FilterOptionsList/index.jsx]: console.log here too.

**Review by halitbatur — COMMENTED** [review]
why is the advanced.js deleted

File: src/components/SelectedFilterOptionList/index.jsx [hunk 3]
```diff
@@ -4,7 +4,22 @@ import "./index.css";
...
+  chosenOptions,
+}) => {
+  const deleteOption = (option) => {
+    const values = chosenOptions[chosenOptionsKey];
+    const index = values.indexOf(option);
+    if (index > -1) {
+      values.splice(index, 1);
+    }
+    // const key = chosenOptions[chosenOptionsKey];
+    // setChosenOptions({ key: values });
```
Comment (khaldoon9kh) [inline src/components/SelectedFilterOptionList/index.jsx]: remove commented code

**Review by khaldoon9kh — CHANGES_REQUESTED** [review]
LGTM other than some console logs and commented parts.

**Comment by fadime-ozdemir** [comment]
> why is the advanced.js deleted
@halitbatur I think we don't need it

**Comment by fadime-ozdemir** [comment]
PTAL

**Comment by fadime-ozdemir** [comment]
PTAL

**Pull request merged**